base/bucket_gocb.go

```diff
@@ -1741,6 +1803,39 @@ func asBool(value interface{}) bool {
 
 }
 
+func asStale(value interface{}) gocb.StaleMode {
```

Comment:
Added a simple test following TestApplyViewQueryOptionsWithStrings which would have broken prior to this fix.